source/content/terminus/13-terminus-4-0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Terminus Guide
```

Comment:
@kporras07 @DuncanSchouten I would prefer to not create a separate page in the nav for this info. This info should just go in the release note

Reply:
Nevermind, I'm going to do some slight rearranging of the sidebar nav